### PR TITLE
Change parser

### PR DESCRIPTION
**Pull request by lingyanhao**
The parser now accepts yyyy-MM-ddTHH:mm and HH:mm formats.

Improvements on the coding style are welcome if you can see a better way to do this.

### CODE REVIEW

**Comment by lingyanhao** [comment]
I'm stacking my PPP changes on this PR

File: docs/UserGuide.adoc [hunk 1]
```diff
@@ -383,7 +384,7 @@ This updates the restaurants to cater for a capacity for 50 customers.
 
 Creates a new booking.
 
-Format: `addBooking c/CUSTOMER_INDEX n/NUMBER OF MEMBERS ts/START_TIME`
+Format: `addBooking c/CUSTOMER_INDEX n/NUMBER OF PERSONS ts/START_TIME`
```
Comment (limli) [inline docs/UserGuide.adoc:387]: persons -> people

File: src/main/java/seedu/address/model/booking/BookingWindow.java [hunk 2]
```diff
@@ -10,28 +13,57 @@
...
     public BookingWindow(LocalDateTime startTime) {
         this.startTime = startTime;
-        this.endTime = startTime.plusHours(1);
+        this.endTime = startTime.plus(DEFAULT_DURATION);
+    }
+
+    /**
+     * Checks if {@code s} is a valid LocalDateTime.
+     */
+    private boolean isValidDateTime(String s) {
```
Comment (limli) [inline src/main/java/seedu/address/model/booking/BookingWindow.java]: reported for 1 length variable name

File: src/main/java/seedu/address/model/booking/BookingWindow.java [hunk 3]
```diff
@@ -10,28 +13,57 @@
...
+            return true;
+        } catch (DateTimeException e) {
+            return false;
+        }
+    }
+
+    /**
+     * Checks if {@code s} is a valid LocalTime.
+     */
+    private boolean isValidTime(String s) {
```
Comment (limli) [inline src/main/java/seedu/address/model/booking/BookingWindow.java]: 1 length variable name

**Review by limli — COMMENTED** [review]
would be good to have ui tests as well

**Pull request merged**